modules/ROOT/pages/custom-response-policy-example.adoc

```diff
@@ -119,26 +134,35 @@ With these premises in mind, we create the YAML file defining the UI of the poli
      allowMultiple: false
 ----
 
-Check the following parameters: `dependsOnKey` and `dependsOnValue` states that the key indicated in `dependsOnKey` must be set to the value indicated in `dependsOnValue` in order to display the fields `responseCondition` or `requestCondition`.
+In the example, the `dependsOnKey` and `dependsOnValue` parameters specify that the key indicated in `dependsOnKey` must be set to the value indicated in `dependsOnValue`, to display the fields `responseCondition` or `requestCondition`.
 
-In the UI, this translates to a checkbox in `evaluateInTheRequest` or `evaluateInTheResponse`, that will make the field `responseCondition` or `requestCondition appear or not depending on the value of the checkbox.
+In the UI, this translates to a checkbox in `evaluateInTheRequest` or `evaluateInTheResponse`, that will make the field `responseCondition` or `requestCondition` appear depending on the value of the checkbox.
 
-For more information about the set up of the YAML file, please refer to xref:custom-policy-4-reference#yaml-configuration-file[YAML Configuration File documentation].
+For more information about how to set up of the YAML file, see xref:custom-policy-4-reference#yaml-configuration-file[YAML Configuration File documentation].
 
-With the previous configuration, the UI will look like something like this:
+Based on the example, the UI is generated as:
 
 image::custom-response-policy-ui.png[custom-response-policy]
 
-And here we can see the usage of `dependsOnKey` and `dependsOnValue` in action:
+The following screenshot illustrates the `dependsOnKey` and `dependsOnValue` parameters:
 
 image::depends-on-key-demo.png[depends-on-key-demo]
 
-==== Implementation
-Now that the input parameters for our policy are defined, we proceed to the implementation.
+[[implment_the_policy]]
+== Implementing the Policy
+
+Now that you have defined the input parameters for your policy, you can start with implementing your policy. To implement your policy, you:
+
+. Add your dependencies in the `pom.xml` file.
+. Add conditions in the `template.xml` file.
 
-First of all, in our `pom.xml` we need to include all the dependencies that we are going to use. In this case, we only need the `http-policy-transform-extension`, that will help us to modify the response information in an easy way.
+=== Add Dependencies in POM.XML File
 
-In the `<dependencies>` section of the policy's pom, include the following dependency:
+In your `pom.xml`, include all the dependencies that you need to use. In this case, you only need the `http-policy-transform-extension`, which will help you to easily modify the response information.
+
+*<Lorena, why do they need to do this? How does it help?>*
```

Comment:
The idea is to show the entire process of developing a custom policy. In this case, as we are using the `http-policy-transform-extension` to simplify the process of transforming the message, we need to include that dependency. I think that is a common use case to include our own extensions as dependencies.

Do you hav any observations about this? Any better approach?

Reply:
I guess what I'm asking is what does adding dependencies do for them. I want to know how it helps them. Like for example when building the house, the foundation is done first. This helps seal the structure of the building. Then we add cement or wood, to create the bare bones of the house, then add fixtures like doors and windows and then do the interiors. I want to break down what does each step translate to for them. Same with the next comment.

Reply:
Oh, ok. Great. We are adding the `http-transform-extension` in order to allow transforming the response (the new status code and the payload). With this extension we are avoiding to write a lot of boilerplate code in order to transform the response message.

I can't exactly map this usage to a nice `user friendly` example, like the one you gave to me with the house.

Reply:
O.K. added! Thanks!